packages/storybook/src/stories/1-MarkdownEditor.stories.js

```diff
@@ -1,9 +1,9 @@
-import React, { useState, useCallback } from 'react';
+import React, { useState, useCallback } from "react";
```

Comment:
Please don't convert all single quotes to double quotes, it makes the PR difficult to review because you can't tell what is a functional change vs a change in code style.

Reply:
I used prettify, I think that converted the quotes, I wasn't aware of these standards.